manage.py

```diff
@@ -6,7 +6,7 @@
 
 def main():
     os.environ.setdefault('DJANGO_SETTINGS_MODULE',
-                          'course_gather.settings.local_settings')
+                          'course_gather.settings.dev_settings')
```

Comment:
I would rather have this remain local settings since that is reserved for local development related configurations IE (inter personal network database servers)

Reply:
Fixed in 664703a.